pppd: Disable wtmp support

Many uClibc based environments lack wtmp and utmp support, therfore remove
the code updating the wtmp information.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

--- a/pppd/sys-linux.c
+++ b/pppd/sys-linux.c
@@ -2267,6 +2267,7 @@ int ppp_available(void)
 
 void logwtmp (const char *line, const char *name, const char *host)
 {
+#if 0
     struct utmp ut, *utp;
     pid_t  mypid = getpid();
 #if __GLIBC__ < 2
@@ -2332,6 +2333,7 @@ void logwtmp (const char *line, const ch
 	close (wtmp);
     }
 #endif
+#endif
 }
 #endif /* HAVE_LOGWTMP */
 
